src/Microsoft.Graph.Core/Microsoft.Graph.Core.csproj

```diff
@@ -19,7 +19,7 @@
     <DelaySign>false</DelaySign>
     <AssemblyOriginatorKeyFile>35MSSharedLib1024.snk</AssemblyOriginatorKeyFile>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
-    <VersionPrefix>1.17.0</VersionPrefix>
+    <VersionPrefix>1.18.0</VersionPrefix>
     <VersionSuffix></VersionSuffix>
     <PackageReleaseNotes>
       - Removed PageIterator. It is now available from the service library.
```

Comment:
Remove old release notes. Add a release note for this work. Will be visible from NuGet.org and the package.

Reply:
Resolved